storage/schemadb/src/lib.rs

```diff
@@ -220,6 +220,20 @@ impl DB {
         Ok(db)
     }
 
+    /// Open db in readonly mode
+    pub fn open_readonly<P: AsRef<Path>>(
+        path: P,
+        cf_opts_map: ColumnFamilyOptionsMap,
+        db_log_dir: &str,
```

Comment:
It makes more sense to me to let `db_log_dir` be a `P`, just like `path`.

Reply:
good point, done.